<launch>
    <arg name="load_calibration" default="false" />
    <arg name="verify_calibration" default="false" />
    <arg name="num_poses" default="20" />

    <node name="aruco_tf" pkg="camera_calib_pkg" type="aruco_tf" output="screen">
        <param name="load_calibration" value="$(arg load_calibration)" />
        <param name="verify_calibration" value="$(arg verify_calibration)" />
        <param name="num_poses" value="$(arg num_poses)" />
    </node>
</launch>